src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
```

Comment:
This can be fixed with css and we can keep the title.  there is no reason the select boxes should take up that much space, so I'm going to fix that 😉